packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -720,6 +742,18 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     )
   }
 
+  private renderLabel(getLabelProps: (options?: GetLabelPropsOptions) => any) {
+    const { label, search } = this.props
+    return Text.create(label, {
+      defaultProps: {
+        as: 'label',
+        className: Dropdown.slotClassNames.label,
+        ...getLabelProps(),
+        ...(!search && { htmlFor: undefined }),
```

Comment:
Downshift adds the `for` as `input` id. Will remove this if there isn't any input. Also this is not really needed, as both input and trigger button have labelledby pointing to this.

Reply:
can you please rephrase this a bit and put as comment in code? looks a bit strange that we need to do it; I'd also encapsulate this in a dedicated method together with spreading `...getLabelProps(),`